### ISSUE TITLE
error during deeplabcut.create_labeled_video

### ISSUE THREAD

**Issue by es1223** [DeepLabCut/DeepLabCut#1374]
Hello again! Thank you for opening this issue.

OS: MaxOS 11.4 Big sur
Anaconda env used:anaconda3
DeepLabCut Version: DeepLabCut2.1.10.4 on Google colab

Describe the problem
After I did labeling my data, I moved all files on Google Drive and used Google colab.
I've got an error when I did deeplabcut.create_labeled_video(path_config_file,videofile_path).
What is a problem?
I attached my google colab note book window and my google drive photo.
![No4](https://user-images.githubusercontent.com/86283566/124203905-b78b7480-db18-11eb-910c-99caf994c927.png)
![No3](https://user-images.githubusercontent.com/86283566/124203911-b9553800-db18-11eb-9e78-bf85d5fb41a6.png)
![No2](https://user-images.githubusercontent.com/86283566/124203912-b9553800-db18-11eb-982f-b578aac0c045.png)
![No1](https://user-images.githubusercontent.com/86283566/124203913-b9edce80-db18-11eb-8574-ac2a00d7d496.png)

Thank you!

**Comment by MMathisLab** [comment]
I don't think create labeled video allows such special characters; @jeylau

**Comment by es1223** [comment]
Thank you again, MMathisLab. 
Which characters do you mean?

**Comment by jeylau** [comment]
She's talking about that odd sequence of characters after "single"
<img width="151" alt="Capture d’écran 2021-07-02 à 1 23 55 PM" src="https://user-images.githubusercontent.com/30733203/124267607-c41fc500-db38-11eb-9b55-b9c74380c75c.png">
Can you confirm that you do have a file "2021-06-05_1_clearbox...h5" (the h5 extension is important) next to your video and the pickle you highlight?

**Comment by es1223** [comment]
Thank you Jeylau.
I’ve got that file after I did deeplabcut.analyze_videos. it was automatically named. Should I change the file name?

and yes. I have a 2021-06-25_1……h5 file.

**Comment by jeylau** [comment]
Can you check in your config.yaml file what you have for "date"? I believe this may be related to https://github.com/DeepLabCut/DeepLabCut/pull/1284, and wrong date formatting on non-English machines. If that is the case, you could reformat it manually and edit your file name so it matches 😌

**Comment by es1223** [comment]
I checked and changed the date in config.yaml. As you commented, date was the wrong format! Thank you ☺

But I get an another error when I did deeplabcut.analyze_videos(path_config_file,videofile_path) after I changed the date.
It is a new error because I didn't get such error in the first problem.
![No5](https://user-images.githubusercontent.com/86283566/124288121-c7906c00-db8b-11eb-9033-50ff0b0f38c2.png)
